packages/expo-camera/src/Camera.types.ts

```diff
@@ -91,6 +91,10 @@ export type CameraCapturedPicture = {
    * Captured image height.
    */
   height: number;
+  /**
+   * The extension of the captured image.
+   */
+  ext: 'jpg' | 'png';
```

Comment:
I’m not really sure about the name of this property. I’d prefer to go with “format” instead. But if you think “ext” works better, go ahead and merge it.

Reply:
Yeah, I was undecided here. format is godd